cocodump/main.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+
+from cocodump.args import args, parse_args
+from cocodump.asm_emitter import emit_asm
+from cocodump.reader import read_img
+from cocodump.target_loader import import_target_decoder, list_target_decoders, normalize_target_name
+
+
+def main():
+    parse_args()
```

Comment:
There is no point to call `parse_args` a second time, because line 3 already does it.

Reply:
Yep